R/pmcmc_control.R

```diff
@@ -124,6 +152,13 @@
 ##'   calculation is a sum of discrete normalised probability
 ##'   distributions, but may not be for continuous distributions!
 ##'
+##' @param n_burnin Optionally, theumber of points to discard as
```

Comment:
```suggestion
##' @param n_burnin Optionally, the number of points to discard as
```

Reply:
updated in 1dfbe21 so that I could patch the generated docs too